prombench/README.md

```diff
@@ -118,3 +118,19 @@ export PR_NUMBER=<PR to benchmark against the selected $RELEASE>
     -v GITHUB_ORG:${GITHUB_ORG} -v GITHUB_REPO:${GITHUB_REPO} \
     -f manifests/prombench/benchmark
 ```
+### Delete and Cleanup
+  - Delete the prombench GKE cluster
```

Comment:
Deleting the cluster does not fall under stopping the benchmark, so this will be misleading.